best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
+(e.g. `CreatePurchase`, `UpdateCreditCard`).
+* Expose one public method on services that does the work
+(i.e. `run`, `call`, or `perform`).
```

Comment:
I often use `#call`, or `#perform` if I'm interacting with a library that wants that (DJ, IIRC), or `#run` if that's the one that makes sense. Not totally sure we need to specify one of these.